From 75d1e77308d7d84547b787a5397d15b11fdf7590 Mon Sep 17 00:00:00 2001
From: Yu Feng <Yu.Feng@windriver.com>
Date: Thu, 21 Jul 2022 15:27:43 +0800
Subject: [PATCH] u-boot-s32: enable CONFIG_XEN_SUPPORT

The patch enables CONFIG_XEN_SUPPORT, so that the U-Boot
can run hypervisor.

Upstream-Status: Pending

Signed-off-by: Yu Feng <Yu.Feng@windriver.com>
---
 configs/aptiv_cvc_sousa_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/aptiv_cvc_sousa_defconfig b/configs/aptiv_cvc_sousa_defconfig
index 3270d7892b..f0a2356aec 100644
--- a/configs/aptiv_cvc_sousa_defconfig
+++ b/configs/aptiv_cvc_sousa_defconfig
@@ -78,3 +78,4 @@ CONFIG_CMD_SMC=y
 CONFIG_FAT_WRITE=y
 CONFIG_MENU=y
 CONFIG_CMD_BOOTMENU=y
+CONFIG_XEN_SUPPORT=y
\ No newline at end of file
-- 
2.35.1

